corehq/apps/reports/models.py

```diff
@@ -302,7 +302,11 @@ def _make_request(self, method, request_name, url, data, requires_sign_in=True):
                 body = json.loads(response.text)
                 return body
         else:
-            error_code = json.loads(response.text)['error']['code']
+            error_code = None
+            if 400 <= response.status_code and response.status_code < 500:
```

Comment:
FYI you can do this:
```suggestion
            if 400 <= response.status_code < 500:
```